razorpay-payments.php

```diff
@@ -357,7 +401,14 @@ protected function createRazorpayOrderId($orderId, $sessionKey)
 
             $data = $this->getOrderCreationData($orderId);
 
-            $razorpayOrder = $api->order->create($data);
+            try
+            {
+                $razorpayOrder = $api->order->create($data);
+            }
+            catch (Exception $e)
+            {
+                echo "Razorpay Error: " . $e->getMessage();
```

Comment:
execution will continue to 413 and raise another exception there, since $razorpayOrder is null.